### PR TITLE
Editted ALS reader to function correctly with ALS 8.3.2 datasets

### PR DESCRIPTION
**Pull request by ttgmichael**
Added ALS 8.3.2 dataset reading capabilities for the new tomopy.

### CODE REVIEW

File: tomopy/io/exchange.py [hunk 1]
```diff
@@ -117,17 +131,79 @@ def read_als_832(fname, ind_tomo=None):
...
+    # xxxx is in incrementals of one, and yyyy is either 0000 or the last projection.
+    # For datasets that take flat while they scan (when the beam fluctuates during scans),
+    # xxxx is always 0000, and yyyy is in intervals given by log file.
+ 
+        if inter_bright == 0:
+            list_flat = tio._list_file_stack(flat_name, ind_flat, digit=4)
+            for x in ind_flat:
+	        body = os.path.splitext(list_flat[x])[0]
+	        ext = os.path.splitext(list_flat[x])[1]
+                body += '_0000'
```
Comment (dgursoy) [inline tomopy/io/exchange.py]: Indentation seems to contain mixed spaces and tabs.

Comment (ttgmichael) [inline tomopy/io/exchange.py]: Apparently they didn't like the way I commented, will fix it tomorrow.

**Comment by ttgmichael** [comment]
Should be good now. I'll check again tomorrow.

**Comment by dgursoy** [comment]
Thanks!

**Pull request merged**